### PR TITLE
chore(deps/octez): bump octez version to be540a2

### PR DESCRIPTION
**Pull request by johnyob**
# Context

# Description

This PR simply bumps the octez version used by the sandbox

# Manually testing the PR

Running the sandbox

### CODE REVIEW

File: crates/octez/src/node.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ impl OctezNode {
             self.octez_node_dir.to_str().expect("Invalid path"),
             "--net-addr",
             http_endpoint,
-            "--local-rpc-addr", // TODO: @alanmarkoTrilitech --local-rpc-addr is not present in future versions of octez-node
+            "--rpc-addr", // TODO: @alanmarkoTrilitech --local-rpc-addr is not present in future versions of octez-node
```
Comment (johnyob) [inline crates/octez/src/node.rs:42]: Remove comment

Comment (alanmarkoTrilitech) [inline crates/octez/src/node.rs:42]: remove todo comment

**Comment by alanmarkoTrilitech** [comment]
note to self - test stopping

**Review by alanmarkoTrilitech — APPROVED** [review]
LGTM!

**Pull request merged**